package org.example.exception;

public class CpfException  extends RuntimeException{

    public CpfException(){
        super("O Cpf n√£o foi informado!");

    }

}
